src/css/components/ButtonGroup.scss

```diff
@@ -18,8 +18,11 @@ $ButtonGroup-space: -1 * $border-width-small;
 // * 1. Overlap elements and override any vertical margins.
 // * 2. Reset `border-radius` so corners line up.
 
+.ButtonGroup > .Button + .Button {
+  margin-left: $ButtonGroup-space; // 1
```

Comment:
The way this was done previously (adding negative right and left margins to all buttons in button group) was overkill. We only need it on one side of adjacent buttons to make it so the border isn't doubled up.